### PR TITLE
BAN-28: Add basic typography styles and their stories

### PR DESCRIPTION
**Pull request by chavlji**
As per [jira ticket](https://reciprocitylabs.atlassian.net/browse/BAN-28)

Screenshot 2020-11-17 at 09 49 52

### CODE REVIEW

File: src/zen-styles/typography.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+h1, h2, h3, h4, h5, h6,
+.h1, .h2, .h3, .h4, .h5, .h6 {
+  margin-bottom: 0.5rem;
+  // font-family: $headings-font-family; TODO: include font
+  font-weight: 500;
```
Comment (mplesec) [inline src/zen-styles/typography.scss:5]: Did designers say why not 400 since that's the default weight?

**Review by mplesec — APPROVED** [review]
Looking really nice 👍

File: src/stories/Typography.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+
+<Meta title="Typography" />
+
+# Typography
+
+Basic headings, paragh, code,...<br />
```
[the remark this answers is not in the document]
Comment (chavlji) [reply src/stories/Typography.stories.mdx]: Typo... 😊

File: src/zen-styles/typography.scss (hunk 1 above)
Comment (chavlji) [reply src/zen-styles/typography.scss]: These things aren't set in stone yet and will have to be revised when designers finish their job.
I just checked it's actually 700 in the current designs so I just bumped it to 700 for now.

**Review by mplesec — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -40,6 +40,7 @@
     "@storybook/addon-links": "~6.0.28",
     "@storybook/addon-notes": "~5.3.21",
     "@storybook/html": "~6.0.28",
+    "@storybook/preset-scss": "^1.0.3",
```
Comment (f-jauregui) [inline package.json:43]: ^ to ~

**Review by f-jauregui — APPROVED** [review]
Looks awesome! 🥇

**Pull request merged**